src/weatherreport/Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: rel deps test
```

Comment:
There should also be steps taken to add the `weatherreport` binary to the built asset, i.e. `make release` should stick `weatherreport` into `rel/couchdb/bin/weatherreport`.

Once this is working I'm happy to contribute the trivial `weatherreport.cmd` we'll need for Windows.

Reply:
The top level `release` target has been modified as requested.